readme.md

```diff
@@ -119,7 +139,7 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
```

Comment:
I think we need a little more context here, i.e. "To save an instance in Rails..."